tables/attributeset.py

```diff
@@ -680,6 +680,139 @@ def __repr__(self):
             return str(self)
 
 
+class ColumnAttributeSet:
+
+    def __init__(self, column):
+
+        self._v_tableattrs = column.table.attrs
```

Comment:
Any particular reason why you don't use access through ``__dict__`` also for this one?